custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml

```diff
@@ -48,7 +49,14 @@ card_media_player:
             if (!variables.ulm_card_media_player_collapsible) {
                 return "12px";
               } else {
-                return entity.state !== "off" ? "12px" : "0px";
+                if (entity.state == "off"){
+                  return "0px";
+                } else if (entity.state == "standby"){
+                  return "0px";
+                } 
+                else {
+                  return "12px";
+                }
```

Comment:
Does this also work?
```suggestion
                return (entity.state === "off" || entity.state === "standby") ? "12px" : "0px";
```

Reply:
I did not work for me. It always applies the 12px padding

Reply:
This fixes it
`return (entity.state === "off" || entity.state === "standby") ? "0px" : "12px";`